sqlair.go

```diff
@@ -300,8 +302,10 @@ func (q *Query) GetAll(sliceArgs ...any) (err error) {
 		sliceVals = append(sliceVals, sliceVal)
 	}
 
+	noRows := true
```

Comment:
[Nit] I think normally you would define the boolean as starting in false (the default value) so that you do not have to negate the name. What about `returnsRows`? Then the check below is: `if !returnsRows && q.pq.HasOutputs`